scripts/enum/animation_strings.lua

```diff
@@ -0,0 +1,107 @@
+-----------------------------------
+-- Animation strings
+-----------------------------------
+xi = xi or {}
+
+--[[
+Looks like in other places we call them entityAnimation and in another for zone parts using a diff packet but the same index (like lamps in psoxja, geysers in dangruf wadi) uses entityVisual
+
+-----------------------------------
+mob:entityAnimationPacket(xi.animationString.STATUS_VISIBLE)
+-----------------------------------
+The base entity is a mob, and anyone in range to see the mob sees the animation.
+We do accept a second argument in this one, for target.
+An example of this is the afflictors in Beadeaux, which target the player. npc:entityAnimationPacket('main', player)
+
+-----------------------------------
+player:entityVisualPacket('saa1')
+-----------------------------------
+In this there is no mob  or npc at all.
+Tt's a part of the zone itself animating, so player is used as base entity and any player that zones in gets sent the packet to light up that part of the zone.
+So the thing animating it's really the zone itself, sort of.
+(There is technically "an object"  or "entity" but its not ID'd like an NPC, and technically speaking a zone dat is laid out like NPCs are part of it anyway)
+
+-----------------------------------
+SendEntityVisualPacket(ID.npc.GEYSER_OFFSET, 'kkj2')
```

Comment:
you probably want to use your shiny new enum instead of the actual string in these examples. Should probably also paraphrase rather than copy that whole discord message, because ol Teo is a long winded pain in the rear that wrote you a novel to point out 3 function calls ^^;;

Reply:
But I liked uncle Teo's wording....

I'll use the enums though and try to make the explanations shorter